lib/Doctrine/DBAL/Platforms/DB2Platform.php

```diff
@@ -107,18 +128,10 @@ public function isCommentedDoctrineType(Type $doctrineType)
      */
     protected function getVarcharTypeDeclarationSQLSnippet($length, $fixed)
     {
-        return $fixed ? ($length ? 'CHAR(' . $length . ')' : 'CHAR(255)')
+        return $fixed ? ($length ? 'CHAR(' . $length . ')' : 'CHAR(254)')
                 : ($length ? 'VARCHAR(' . $length . ')' : 'VARCHAR(255)');
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    protected function getBinaryTypeDeclarationSQLSnippet($length, $fixed)
-    {
-        return $fixed ? 'BINARY(' . ($length ?: 255) . ')' : 'VARBINARY(' . ($length ?: 255) . ')';
```

Comment:
Most likely, binary data can be safely stored in "char" fields in DB2. For now, just disabling the type since it's broken anyways.